custom_typings/plylog.d.ts

```diff
@@ -3,10 +3,10 @@ declare module 'plylog' {
   class PolymerLogger {
     constructor(options: any)
     setLevel(newLevel: string)
-    error(message: string, metadata?: any)
-    warn(message: string, metadata?: any)
-    info(message: string, metadata?: any)
-    debug(message: string, metadata?: any)
+    error(...data: any[])
```

Comment:
as discussed, the winston logger is very lax. We'll keep the stronger argument definitions for our own sake.